lib/node_modules/@stdlib/stats/base/dists/negative-binomial/cdf/benchmark/benchmark.js

```diff
@@ -32,18 +33,26 @@ var cdf = require( './../lib' );
 // MAIN //
 
 bench( pkg, function benchmark( b ) {
+	var len;
 	var r;
 	var p;
 	var x;
 	var y;
 	var i;
 
+	len = 100;
+	x = new Float64Array( len );
+	r = new Float64Array( len );
+	p = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( 0, 100.0 );
```

Comment:
```suggestion
		x[ i ] = uniform( 0.0, 100.0 );
```


Applies throughout the entire PR.

Reply:
This change is still needed.